src/Lykke.Tools.UserWalletBalanceReport/Program.cs

```
@@ -131,74 +146,89 @@ private static async Task Execute(string settingsFilePath)
                     clientIds = await File.ReadAllLinesAsync(settings.CurrentValue.ClientIdsFilePath);
                 }
 
+                // If AssetId is omitted, IncludeZeroBalances should be false.
+                var includeZeroBalances = settings.CurrentValue.IncludeZeroBalances
```

better to throw an exception if configuration is invalid